src/I18n/DateFormatTrait.php

```diff
@@ -130,7 +130,7 @@ protected function _formatObject(
      * @param \DateTimeZone|string|null $tz The timezone for the instance
      * @return static|null
      */
-    protected static function _parseDateTime(
+    protected static function doParseDateTime(
```

Comment:
`parseDateTime` also exists on the `I18n\DateTime` class, therefore this needed to be different so it doesn't overwrite it.